### ISSUE TITLE
Adding Dropout Layer to models and accessing nn layers information

### ISSUE THREAD

**Issue by Elimane0800** [ollama/ollama-python#382]
Is it possible to add dropout layer to the models (to enable Monte Carlo dropout for example) ? Plus access the neural network layer information of models. 

Elimane

**Comment by ParthSareen** [comment]
Ollama is just a runner for the models. It seems you're looking for running with PyTorch or something alike?
